DependencyInjection/Configuration.php

```diff
@@ -24,6 +24,144 @@ public function __construct($rootName)
     {
         $this->rootName = $rootName;
     }
+    
+    private $defaultFormTypes = array(
+      'doctrine_orm' => array(
```

Comment:
@tobias-93 please copy/paste the arrays from below (with comments and indentation preserved)